GoogleForCreators/web-stories-wp
# Redesign: Shape library buttons

## Context



## Summary
Redesign:
- Adjust general styles and margins
- Use roving index for the buttons
- Use focus outline
- Adds onClick handler to the shape button separately for enabling default keyhandlers.


## Relevant Technical Choices



## To-do




## User-facing changes

| Before | After |
|-------|-------|
| <img width="361" alt="Screenshot 2021-03-15 at 10 28 12" src="https://user-images.githubusercontent.com/3294597/111124538-e7fed100-8570-11eb-8e22-1d75150b7bc3.png"> | <img width="361" alt="Screenshot 2021-03-15 at 09 31 14" src="https://user-images.githubusercontent.com/3294597/111124584-f2b96600-8570-11eb-9318-a5dbf8d60d12.png"> |
| N/A | <img width="364" alt="Screenshot 2021-03-15 at 09 31 26" src="https://user-images.githubusercontent.com/3294597/111124617-fd73fb00-8570-11eb-8401-a3a7afaaf0c6.png"> |




## Testing Instructions
1. Open the Shape library
2. Verify the style looks like in Figma
3. Tab into the shape library
4. Verify you can move around using the arrow keys and add a shape by clicking "Enter" or "Space".


### QA
Use the steps from the "Testing instructions"





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #6215
